scrape/scrape.go

```diff
@@ -1418,8 +1418,16 @@ type appendErrors struct {
 }
 
 func (sl *scrapeLoop) append(app storage.Appender, b []byte, contentType string, ts time.Time) (total, added, seriesAdded int, err error) {
+	p, warning := textparse.New(b, contentType)
+	if warning != nil {
+		level.Warn(sl.l).Log(
```

Comment:
Even if it is warning, this should be logged as debug, because it would be too verbose.

Reply:
fixed, thanks